bower.json

```diff
@@ -1,17 +1,20 @@
 {
   "name": "ember-fetch",
   "dependencies": {
-    "ember": "1.11.1",
-    "ember-cli-shims": "ember-cli/ember-cli-shims#0.0.3",
-    "ember-cli-test-loader": "ember-cli-test-loader#0.1.3",
-    "ember-data": "1.0.0-beta.16.1",
-    "ember-load-initializers": "ember-cli/ember-load-initializers#0.1.4",
-    "ember-qunit": "0.3.1",
-    "ember-qunit-notifications": "0.0.7",
-    "ember-resolver": "~0.1.15",
-    "jquery": "^1.11.1",
-    "loader.js": "ember-cli/loader.js#3.2.0",
-    "qunit": "~1.17.1",
+    "ember": "components/ember#release",
```

Comment:
any reason why this isn't a fixed version? ember-try should handle release channels